share/models/change.py

```diff
@@ -251,7 +251,11 @@ def _resolve_change(self):
             elif isinstance(v, dict):
                 inst = self._resolve_ref(v)
                 change[k] = inst
-                change[k + '_version'] = inst.version
```

Comment:
It could be an `isinstance` check, but I was trying to be more pythonic and duck-type it.